scripts/build.py

```diff
@@ -334,17 +419,30 @@ def build_minified_third_party_libs():
 
     print 'Building minified third party libs...'
 
-    third_party_js = os.path.join(
+    STAGING_THIRD_PARTY_JS = os.path.join(
         THIRD_PARTY_GENERATED_STAGING_DIR, 'js', 'third_party.min.js')
-    third_party_css = os.path.join(
+    STAGING_THIRD_PARTY_CSS = os.path.join(
         THIRD_PARTY_GENERATED_STAGING_DIR, 'css', 'third_party.min.css')
-    fonts_dir = os.path.join(THIRD_PARTY_GENERATED_STAGING_DIR, 'fonts', '')
+    STAGING_FONTS_DIR = os.path.join(
+        THIRD_PARTY_GENERATED_STAGING_DIR, 'fonts', '')
 
     dependency_filepaths = get_dependencies_filepaths()
-    _minify_and_create_sourcemap(dependency_filepaths['js'], third_party_js)
-    _join_files(dependency_filepaths['css'], third_party_css)
-    _minify(third_party_css, third_party_css)
-    _copy_fonts(dependency_filepaths['fonts'], fonts_dir)
+    _minify_and_create_sourcemap(
+        dependency_filepaths['js'], STAGING_THIRD_PARTY_JS)
+    if not does_directory_tree_exist(STAGING_THIRD_PARTY_CSS):
```

Comment:
There's a lot of stuff clumped together here, it's a bit hard to read.

Let's put newlines between the JS, CSS and fonts parts.

Reply:
Added newline after declaring variables.